packages/core/src/lib/GooglePlay.ts

```diff
@@ -58,4 +61,47 @@ export class GooglePlay {
     await this.gradleWrapper.executeGradleCommand(
         ['publishBundle', '--artifact-dir', filepath, '--track', track]);
   }
+
+  /**
+   * Connects to the Google Play Console and retrieves a list of all Android App Bundles for the
+   * given packageName. Finds the largest versionCode of those bundles and returns it. Considers
+   * both ChromeOS and Android Releases.
+   */
+  async getLargestVersionCode(
+      packageName: string,
+      serviceAccountJsonFilePath: string,
+  ): Promise<number> {
+    if (!this._googlePlayApi) {
+      this._googlePlayApi = this.getAndroidClient(serviceAccountJsonFilePath);
+    }
+    const edit = await this._googlePlayApi.edits.insert({packageName: packageName});
+    const editId = edit.data.id!;
+    const bundleResponse =
+      await this._googlePlayApi.edits.bundles.list({packageName: packageName, editId: editId});
+    const versionCode = Math.max(...bundleResponse.data.bundles!!.map(bundle => bundle.versionCode!!));
+    // cleanup
+    await this._googlePlayApi.edits.delete({editId: editId, packageName: packageName});
+
+    return versionCode;
+  }
+
+  /**
+   * This fetches the Android client using the bubblewrap configuration file.
+   */
+  private getAndroidClient(
+      serviceAccountJsonFilePath: string,
+  ): androidpublisher_v3.Androidpublisher {
+    // Initialize the Google API Client from service account credentials
+    const jwtClient = new google.auth.JWT({
+      keyFile: serviceAccountJsonFilePath,
+      scopes: ['https://www.googleapis.com/auth/androidpublisher'],
+    },
```

Comment:
nit: I don't think we need this comma.

Reply:
The linter wants a comma there.

Reply:
Yep, our linter requires dangling commas. That makes diffs nicer when adding new items to the parameters.